data/net/minecraft/server/MinecraftServer.mapping

```diff
@@ -90,9 +94,9 @@ CLASS net/minecraft/server/MinecraftServer
 	METHOD kickUnlistedPlayers (Lnet/minecraft/commands/CommandSourceStack;)V
 		ARG 1 commandSource
 	METHOD logChatMessage (Lnet/minecraft/network/chat/Component;Lnet/minecraft/network/chat/ChatType$Bound;Ljava/lang/String;)V
-		ARG 1 content
+		ARG 1 message
 		ARG 2 boundChatType
-		ARG 3 header
+		ARG 3 sender
```

Comment:
These are not the same. Needs investigating.

Reply:
The original parameter names are correct.